ubyssey/static/src/styles/modules/_header.scss

```diff
@@ -166,7 +166,7 @@ nav.main {
 header.topbar, header.mobile {
   width: 100%;
   z-index: $z-scale-2;
-  position: fixed;
+  position: relative;
   top: 0;
```

Comment:
The mobile header seems to be missing its bottom border:

![screen shot 2017-08-21 at 12 17 40 pm](https://user-images.githubusercontent.com/2547035/29534410-ce2411e0-866a-11e7-95d4-1d2d43191e52.png)